src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/catalog.cljs

```diff
@@ -181,6 +179,12 @@
    :component-did-mount
    (fn [{:keys [props state refs]}]
      (let [{:keys [library-schema questions]} props]
+        (let [versions (get-in library-schema [:versions])]
+          (mapv (fn [version]
+                  (let [currentVersion (get-in library-schema [:properties (keyword version) :current])]
+                    (utils/log currentVersion)
+                    (swap! state update :attributes assoc (keyword version) currentVersion))) ;;js/parseInt should work?
```

Comment:
actually ... this entire `let` block. Are these version numbers anywhere? (they could be, I'm just not seeing it)